backoffice/www/handlers/login.py

```diff
@@ -0,0 +1,21 @@
+from backoffice.www.handlers.base import BaseBOHandler
+
+
+class LoginHandler(BaseBOHandler):
+
+  ROUTE= "/login"
+
+  def get(self):
+    self.write('<html><body><form action="/login" method="post">'
+               'Name: <input type="text" name="name">'
+               '<input type="submit" value="Sign in">'
+               '</form></body></html>')
+
+  def post(self):
+    self.set_secure_cookie("user", self.get_argument("name"))
```

Comment:
so users can always login? We don't even check if the user exists?

Fine for me, it's more out of curiosity; if this is the case, we don't even need a login handler. I guess the server will be running under a service that is only reachable by a few people who have the IP whitelisted in Amazon. I think this is a reasonable way to start.

Reply:
Hmmm I'm out of the loop but that might warrant confirmation by @tehf0x and/or @olivierteboul?